sdks/java/harness/src/main/java/org/apache/beam/fn/harness/control/ProcessBundleHandler.java

```diff
@@ -1021,13 +991,12 @@ void shutdown() throws Exception {
   public abstract static class BundleProcessor {
     public static BundleProcessor create(
         Cache<Object, Object> processWideCache,
-        InMemory bundleProgressReporterAndRegistrar,
+        BundleProgressReporter.InMemory bundleProgressReporterAndRegistrar,
```

Comment:
I see, there are two "InMemory" which are subclass of different interface. If so then may remove the import of `org.apache.beam.fn.harness.control.BundleProgressReporter.InMemory` and also change all other raw InMemory to `BundleProgressReporter.InMemory` in this file

Reply:
Thanks I missed this one.

Fixed.